.mergify.yml

```diff
@@ -56,7 +56,8 @@ pull_request_rules:
       - -closed
       - author=mergify[bot]
       - check-success=CLA
-      - schedule=Mon-Mon 14:00-16:00[Europe/Paris]
+      - schedule=Mon-Mon 12:00-13:00[Europe/Paris]
+      - updated-at<2 days ago
```

Comment:
![Uploading image.png…]()

Reply:
With this new approach, notifications will have up to 9 days of delay, if I did make the right formula:
7 days + 2 days

Reply:
```suggestion
      - updated-at<2 days ago
      - "#assignee>=1"
```